integration/test-integration/src/test/resources/dockerCompose/backend-service-tls.yaml

```
@@ -1,6 +1,6 @@
       - "5006:5006"
   mockBackend:
-    image: wso2/choreo-connect-mock-backend:1.1.0-m2-SNAPSHOT
+    image: wso2/choreo-connect-mock-backend:1.1.0-alpha2-SNAPSHOT
```

Even for tests, docker images should not be snapshots
```suggestion
    image: wso2/choreo-connect-mock-backend:1.1.0-alpha2
```